src/themes/masschallenge/modules/modal.variables

```diff
@@ -0,0 +1,9 @@
+/********** colors **********/
+
+@textColor: @itemTextColor;
+@textareaBackground: #F1F3F4;
+@textareaColor: #8D8F94;
+@listContentColor: @midGray;
+@borderColor: @textareaBackground;
```

Comment:
Re my previous comment about color naming- this feels like a better convention as it describes each variable as being for a `color + object`.